### PR TITLE
Feature/ees 322 release type in content api

### PR DESCRIPTION
**Pull request by negatron99**
Added tests for the front-end to prove it working, tests for the logo showing, content showing and the negative cases.

### CODE REVIEW

File: src/explore-education-statistics-frontend/src/modules/find-statistics/__tests__/PublicationReleasePage.test.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  test('renders national statistics section', () => {
+    const { container } = render(
+      <PublicationReleasePage
+        publication=""
+        release=""
+        data={(NationalStats as unknown) as Release}
+      />,
+    );
+
+    // @ts-ignore
```
Comment (duncan-at-hiveit) [inline src/explore-education-statistics-frontend/src/modules/find-statistics/__tests__/PublicationReleasePage.test.tsx]: Can this be written as:

```
    const elements = container.querySelectorAll('.govuk-accordion__section-header h3 button') as Element[],
```

without the need for the "ts-ignore"?

File: src/explore-education-statistics-frontend/src/modules/find-statistics/__tests__/PublicationReleasePage.test.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  test('renders official statistics section', () => {
+    const { container } = render(
+      <PublicationReleasePage
+        publication=""
+        release=""
+        data={(OfficialStats as unknown) as Release}
+      />,
+    );
+
+    const elements = [
```
Comment (duncan-at-hiveit) [inline src/explore-education-statistics-frontend/src/modules/find-statistics/__tests__/PublicationReleasePage.test.tsx]: Same here...

File: src/explore-education-statistics-frontend/src/modules/find-statistics/__tests__/PublicationReleasePage.test.tsx (hunk 1 above)
Comment (negatron99) [reply src/explore-education-statistics-frontend/src/modules/find-statistics/__tests__/PublicationReleasePage.test.tsx]: The value returned from querySelectorAll doesn't have the find method on it, it's not an array.

All the elements are added to an array using the array destructuring operator, but our TypeScript configuration doesn't like this, even though it's valid to do it this way.

File: src/explore-education-statistics-frontend/src/modules/find-statistics/__tests__/PublicationReleasePage.test.tsx (hunk 2 above)
Comment (negatron99) [reply src/explore-education-statistics-frontend/src/modules/find-statistics/__tests__/PublicationReleasePage.test.tsx]: As above

**Pull request merged**